pyresample/geometry.py

```diff
@@ -440,6 +441,19 @@ def __init__(self, lons, lats, nprocs=1):
         elif lons.ndim != 2:
             raise ValueError('2 dimensional lon lat grid expected')
 
+def get_array_hashable(arr):
+    """Compute a hashable form of the array `arr`.
+
+    Works with numpy arrays, dask.array.Array, and xarray.DataArray.
+    """
+
+    if isinstance(arr, DataArray) and np.ndarray != DataArray: # look for precomputed value
```

Comment:
`np.ndarray is not DataArray`?

Reply:
That's because of the ugly import error trick in the beginning.

Reply:
Yes, but `DataArray is not np.ndarray` is more accurate for what you are testing (and faster).